postgres/datadog_checks/postgres/statements.py

```diff
@@ -334,10 +344,32 @@ def _emit_pg_stat_statements_metrics(self):
         except psycopg2.Error as e:
             self._log.warning("Failed to query for pg_stat_statements count: %s", e)
 
+    def _emit_pg_stat_statement_query_text_length_metrics(self, rows):
+        txt_length_sum = 0
+        txt_length_max = 0
+        for r in rows:
+            query_length = len(r['query'])
+            txt_length_sum += query_length
+            if query_length > txt_length_max:
+                txt_length_max = query_length
+        self._check.gauge(
```

Comment:
Would this one make more sense as a histogram? I think that if we just track the query text length for each row, we'd have the count, max and average, right?